ios/RNSentry.m

```diff
@@ -62,7 +62,14 @@ + (BOOL)requiresMainQueueSetup {
         return event;
     };
 
-    [options setValue:beforeSend forKey:@"beforeSend"];
+    NSMutableDictionary * mutableOptions =[options mutableCopy];
+    [mutableOptions setValue:beforeSend forKey:@"beforeSend"];
```

Comment:
`l`:
```suggestion
    NSMutableDictionary * mutableOptions = [options mutableCopy];
    mutableOptions[@"beforeSend"] = beforeSend;
```

Reply:
I'd rather keep as it is and change in another PR, there are more places in this class doing the same way